Swiftfin/Views/VideoPlayer/VLCPlayerViewController.swift

```diff
@@ -237,22 +261,49 @@ class VLCPlayerViewController: UIViewController {
 		label.alpha = 0
 		label.translatesAutoresizingMaskIntoConstraints = false
 		label.font = .systemFont(ofSize: 48)
+		label.layer.zPosition = 1
 		return label
 	}
 
-	@objc
-	private func didTap() {
-		didGenerallyTap()
+	// MARK: GestureLockedOverlayView
+
+	private func makeGestureLockedOverlayView() -> UIView {
+		let backgroundView = UIView()
+		backgroundView.layer.zPosition = 1
+		backgroundView.alpha = 0
+		backgroundView.translatesAutoresizingMaskIntoConstraints = false
+		let button = UIButton(type: .custom, primaryAction: UIAction(handler: { [weak self] _ in
+			self?.isGesturesLocked = false
+			self?.hideLockedOverlay()
+			self?.didGenerallyTap()
+		}))
+		button.translatesAutoresizingMaskIntoConstraints = false
+		button.setImage(UIImage(systemName: "lock.open", withConfiguration: UIImage.SymbolConfiguration(pointSize: 48))?
```

Comment:
I think just "lock", or "lock.circle.fill" would be a better indicator for when gestures are locked.